### ISSUE TITLE
Keys management

### ISSUE THREAD

**Issue by Dahlgren** [Dahlgren/arma-server-web-admin#117]
* Show all current keys in keys directory
* Copy (all) keys from mod to keys directory

**Comment by Varrkan82** [comment]
It would be better to create orremove the symbolic links on all keys in mod's "key" directory to server's "keys" directory at mod swithcing ON or OFF

**Comment by Dahlgren** [comment]
Yes, that is probably how it has to works to support different keys on multiple different configurations

**Comment by Freddo3000** [comment]
Perhaps this could be of use? https://gist.github.com/Freddo3000/a5cd0494f649db75e43611122c9c3f15

I manage keys and mods using that script, where it symlinks the keys in order to keep them up to date.

**Comment by Dahlgren** [comment]
Implementing keys management is quite easy. Maintaining virtual server folders across Windows and Linux is not so much fun.

If you want to try it out you can use the git branch `feature/virtual-server-folder`. I've only tested it on Windows (as it's the annoying platform in terms of symlinks) and it currently requires Node 10+.

It adds an extra burden on Windows as the server binary gets a new path which triggers a new entry in Windows Firewall that must be permitted.
